src/classes/CMT_MetadataAPI.cls

```diff
@@ -99,6 +99,15 @@ public class CMT_MetadataAPI {
      * @param mtToDeploy List of CustomMetadata objects to deploy
      */
     public static void deployMetadata(String jobId, List<Metadata.CustomMetadata> mtToDeploy) {
+        deployMetadata(jobId, mtToDeploy, null);
+    }
+
+    /**
+     * @description Initiate a deployment of custom metadata type records.
+     * @param jobId Unique string to act as a "jobId" when writing the final deployment status to the Custom Settings
+     * @param mtToDeploy List of CustomMetadata objects to deploy
+     */
+    public static void deployMetadata(String jobId, List<Metadata.CustomMetadata> mtToDeploy, Boolean isHelpTextRelated) {
```

Comment:
@ptewson-sfdo - Currently overloading this method so I could add a flag for handleResult() in line 162. Flag is for determine which Custom Setting to update, CRLPS or Data Import. Any thoughts on a better way to handle the custom settings update in that callback/result handler?